### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Allicolyer**
shiny new documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,50 +24,53 @@ Recurse Roulette requires users to authenticate with Recurse Center before signi
...
-The sever exposes three endpoints for the client to use:
+Recurse Roulette shuffles and reconnects clients every five minutes. To keep the clock consistent across all clients, the server keeps track of the time until the next chat round. Clients request the time from the server as soon as they connect.
+
+Clients need to click "Join" to connect to the signaling server and to be paired for the next chat round. When this happens, the client is given a peer ID by the server. When a new round starts, each client that has joined requests a partner from the server. The server pairs each client as the requests come in. If there is another client that has requested a partner, the server pairs those two clients, otherwise the server remembers the client’s peer ID until another client requests a partner. When the round ends, each client's peer ID is destroyed, which ends the current call, and each client is given a new ID for the next round. If a client disconnects, its peer ID is destroyed, and it must rejoin to receive a new one.
+
+The sever exposes three endpoints:
```
Comment (georgemandis) [inline README.md]:
```suggestion
The server exposes three endpoints:
```

**Review by georgemandis — CHANGES_REQUESTED** [review]
Looks good to me, except for that one typo!

File: README.md (hunk 1 above)
Comment (Allicolyer) [reply README.md]: sever is what I meant!

Comment (georgemandis) [reply README.md]: You're right. A sever would probably expose at least three endpoints. A severe sever? Possibly more!

**Comment by Allicolyer** [comment]
yippie! Did you notice the updated gif?

**Pull request merged**